formbricks/formbricks
# feat: survey error modal/page

## What does this PR do?

Adds an error page and error modal for link surveys and in-app surveys respectively for when submission of responses have failed more than the number of retries alloted. The error page/modal shows all the respones from the survey and has two button, Copy all response as json, and send to email. 

Fixes #1082

https://www.loom.com/share/a04569b18f8e4bf8bcf402f9bfb2611e

## Type of change




## How should this be tested?

Create a link survey or in-app survey, Mock response submission failure in responseQueue.ts, by fixing success to false on line 46. Complete the survey and wait 3 seconds after the thank you page is shown. The error page should appear

## Checklist

### Required


### Appreciated

- [] Updated the Formbricks Docs if changes were necessary
